Disable use of execinfo backtrace because it causes a crash.

Upstream bug report: https://tracker.ardour.org/view.php?id=8211
--- a/libs/pbd/wscript	2020-06-10 14:33:53.822160367 +0200
+++ b/libs/pbd/wscript	2020-06-10 14:34:03.742215341 +0200
@@ -109,7 +109,6 @@
     autowaf.check_pkg(conf, 'glibmm-2.4', uselib_store='GLIBMM', atleast_version='2.32.0', mandatory=True)
     autowaf.check_pkg(conf, 'giomm-2.4', uselib_store='GIOMM', atleast_version='2.2', mandatory=True)
 
-    conf.check(header_name='execinfo.h', define_name='HAVE_EXECINFO',mandatory=False)
     conf.check(header_name='unistd.h', define_name='HAVE_UNISTD',mandatory=False)
     if not Options.options.ppc:
         conf.check_cc(
